private/CreateTaskParameters.m

```diff
@@ -136,6 +136,8 @@
 TaskParameters.GUI.LeftPokeAttenPrcnt = 73;
 TaskParameters.GUI.CenterPokeAttenPrcnt = 95;
 TaskParameters.GUI.RightPokeAttenPrcnt = 73;
+TaskParameters.GUI.LeftSoundAttenPrcnt = 80;
+TaskParameters.GUI.RightSoundAttenPrcnt = 80;
```

Comment:
This is still missing few thins:
1. Since you are adding a new GUI element, you need to bump up the GUI version counter by one: https://github.com/ioannapandi/Mouse2AFC/pull/2/files#diff-7cff12fc600b71830f25a6786325a490R13
2. You've created the Variable but you didn't actually show it, to show it you need to add the following code below:
```MATLAB
TaskParameters.GUIPanels.LightIntensity = {'LeftPokeAttenPrcnt','CenterPokeAttenPrcnt','RightPokeAttenPrcnt','StimAfterPokeOut', 'BeepAfterMinSampling'};
TaskParameters.GUIPanels.SoundDiscremenation = {'LeftSoundAttenPrcnt', 'RightSoundAttenPrcnt'}
TaskParameters.GUIPanels.StimulusSelection = {'OmegaTable','TableNote','BetaDistAlphaNBeta','StimulusSelectionCriteria','LeftBias','LeftBiasVal','CorrectBias'};
``` 
Then below [here](https://github.com/ioannapandi/Mouse2AFC/pull/2/files#diff-d130cd7064a46e44fa26d39708b58ad5R232) you need to add the new panel:
```MATLAB
TaskParameters.GUITabs.Sampling = {'CurrentTrial','LightIntensity','Auditory','Sampling','StimulusSelection',...
                                   'SoundDiscremenation'};
```

Reply:
1. I did it in my code but I thought it is better to add the value after the review process 
2. I had the following code in my previous local files but I forgot to update it today. 
TaskParameters.GUIPanels.SoundDiscrimination = {'LeftSoundAttenPrcnt','RightSoundAttenPrcnt'}
TaskParameters.GUITabs.Sampling = {'CurrentTrial','LightIntensity','Auditory','SoundDiscrimination','Sampling','StimulusSelection'};
We can just correct the word Discrimination in your code and it's the same

Reply:
> 1. I did it in my code but I thought it is better to add the value after the review process

I'd say that you should always include the whole code, if the reviewer has comments he won't let the code through except when the changes are first implemented. Otherwise, if not all the code is included, then it's difficult to judge whether it was forgotten or not yet included.

> I had the following code in my previous local files but I forgot to update it today. ...

I can't judge the order here, I must test it locally to see how it looks in the GUI.

Reply:
Just had this thought now: how about renaming `LeftSoundAttenPrcnt ` to `LeftSpeakerAttenPrcnt  (and same to right-speajer)

Reply:
I agree! I changed it!